sdlib/d/gc/thread.d

```diff
@@ -126,19 +127,18 @@ private:
 	Mutex mThreadList;
 	ThreadRing registeredThreads;
 
-	Mutex stopTheWorldMutex;
+	shared SharedLock createThreadLock;
 
 public:
 	/**
 	 * Thread management.
 	 */
 	void enterThreadCreation() shared {
```

Comment:
This isn't about creating thread anymore. This is about not stopping the world.

Reply:
ok.